src/Microsoft.ApplicationInsights/TelemetryClient.cs

```diff
@@ -29,13 +29,15 @@ public sealed class TelemetryClient
 #endif
         private readonly TelemetryConfiguration configuration;
         private string sdkVersion;
-        
+
+#pragma warning disable 612, 618
         /// <summary>
         /// Initializes a new instance of the <see cref="TelemetryClient" /> class. Send telemetry with the active configuration, usually loaded from ApplicationInsights.config.
         /// </summary>
         public TelemetryClient() : this(TelemetryConfiguration.Active)
```

Comment:
Should we also obsolete this constructor (since it implicitly uses TC.Active) ?
i.e. DI users should always use an injected TC.

Reply:
(by TC I mean TelemetryConfiguration)

Reply:
yes! definitely!